entrypoint.py

```diff
@@ -84,6 +85,7 @@ def main() -> None:
     if os.getenv("GITHUB_ACTIONS") is None:
         os.environ["INPUT_OTHER_NAMES"] = "lint\npkg"
         os.environ["INPUT_MIN_PYTHON"] = "3.8"
+        os.environ["INPUT_MAX_PYTHON"] = "3.12-dev"
```

Comment:
It no longer works like before, I did run the entrypoint directly and it produced output only for py38-py311 --- without py312 among them, something I still want to be enabled by default. Also the param for maxpython is not the runner name and should only be the "3.12" value, even if internally we do install "3.12-dev" as there is no release.